fusion/cone_pos[] cone_detected
string color
float32 mean_x
float32 mean_y
float32 update_round
float32 update_time
